clang_delta/RemoveUnusedStructField.cpp

```diff
@@ -280,6 +280,7 @@ void RemoveUnusedStructField::getInitExprs(const Type *Ty,
 
     if (FD == TheFieldDecl) {
       InitExprs.push_back(Init);
+      //TODO: Is the (void)VecSz necessary?
       TransAssert((VecSz == 1) && "Bad IndexVector size!"); (void)VecSz;
```

Comment:
Yes, (void)VecSz is needed for avoiding unused-var-warnings issued by the compiler when TransAssert is nullified. 

Can you remove the TODO comment?